src/main.c

```diff
@@ -164,11 +171,13 @@ main(int argc,
       theight = cheight * opt.thumb / 100;
     }
 
+    imlib_context_set_anti_alias(1);
     thumbnail =
-      gib_imlib_create_cropped_scaled_image(image, 0, 0, cwidth, cheight,
-                                            twidth, theight, 1);
-    if (thumbnail == NULL)
-      gib_eprintf("Unable to create scaled Image\n");
```

Comment:
There are some replacements for `gib_eprintf` that don't exactly match its function.

```C
gib_eprintf: print error message and exit
gib_weprintf: print warning message and continue
```

Reply:
Thank you; I added exits to the two places I removed eprintf without adding the exit